domain/induction/src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/induction/dto/UpdateFutureWorkInterestsDto.kt

```diff
@@ -4,7 +4,7 @@ import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.Wor
 import java.util.UUID
 
 data class UpdateFutureWorkInterestsDto(
-  val reference: UUID,
+  val reference: UUID?,
```

Comment:
Oops, I should have foreseen this! If an object within the domain is null initially, then it will be impossible to provide a reference in the update request 🤦‍♂️

Reply:
👍 , yes, and thats also consistent with what we do in classes like `UpdateStepDto` for the same reason. Nice 👍